.env.prd.json

```
@@ -1,8 +1,8 @@
 {
   "ENV": "production",
-  "CLIENT_ID": "apimock",
-  "CLIENT_SECRET": "apimock_secret",
-  "BASE_URL": "https://apimock.lazycatlabs.com",
+  "CLIENT_ID": "base_auth_app",
+  "CLIENT_SECRET": "base_auth_secret_789",
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The "CLIENT_SECRET": "base_auth_secret_789", key has incorrect delimiter

(IncorrectDelimiter)